assets/src/js/pro/blocks-v3/components/block-edit.js

```diff
@@ -362,9 +416,13 @@ export const BlockEdit = ( props ) => {
 
 	// Listen for validation error events from other blocks
 	useEffect( () => {
-		const handleErrorEvent = () => {
-			lockPostSaving( clientId );
-			setShowValidationErrors( true );
+		const handleErrorEvent = ( event ) => {
+			// Only handle if this event is for this specific block
+			if ( clientId === event.detail.acfBlockWithValidationErrors ) {
```

Comment:
The event passed by the hanlder seems to be called `acfBlocksWithValidationErrors`, in plural.
```suggestion
			if ( clientId === event.detail.acfBlocksWithValidationErrors ) {
```

Reply:
Fixed. Thanks!